### DISCUSSION TITLE
Flyte community user badges

### DISCUSSION THREAD

**Discussion by cameronraysmith** [flyteorg/flyte#5111] [Ideas]
Is there a community-recommended way to include badges in the docs of repositories that use Flyte?

I did not see one in the docs or

- https://github.com/flyteorg/flyte/issues?q=is%3Aissue+is%3Aopen+badges
- https://github.com/flyteorg/flyte/discussions?discussions_q=is%3Aopen+badges

Here is one proposal

[![flyte-badge]][flyte-link]

[flyte-badge]: https://img.shields.io/badge/Flyte-Carrier-gray.svg?color=7552A2&logo=data:image/svg%2bxml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB2aWV3Qm94PSIwIDAgMzYwIDMzMy40ODM3OSI+PGRlZnM+PHN0eWxlPi5jbHMtMXtmaWxsOnVybCgjbGluZWFyLWdyYWRpZW50KTt9PC9zdHlsZT48bGluZWFyR3JhZGllbnQgaWQ9ImxpbmVhci1ncmFkaWVudCIgeDE9IjYuMTkzNjUiIHkxPSIxNjcuMzQ3NDUiIHgyPSIzNTIiIHkyPSIxNjcuMzQ3NDUiIGdyYWRpZW50VW5pdHM9InVzZXJTcGFjZU9uVXNlIj48c3RvcCBvZmZzZXQ9IjAiIHN0b3AtY29sb3I9IiNlNWM5ZDMiLz48c3RvcCBvZmZzZXQ9IjAuMzMiIHN0b3AtY29sb3I9IiNiMzg1ZGYiLz48c3RvcCBvZmZzZXQ9IjEiIHN0b3AtY29sb3I9IiM2ZjJhZWYiLz48L2xpbmVhckdyYWRpZW50PjwvZGVmcz48cGF0aCBjbGFzcz0iY2xzLTEiIGQ9Ik05MC45NjQ3NSwxNzAuODEzMzRjNy42Mjc3Ny0xOC4xMjcsMTEuNTEwNDctMzYuNzc2NTUsMTUuMDA2NC01NS40OTI4OSw1Ljk2MDM3LTMxLjkxMDIsOS45OS02NC4wMzIzLDguNjM2MjEtOTYuNTk0NzUtLjI1NzIzLTYuMTg3NDIsMS40ODQ5NS04LjQ4NzYzLDguMTQ5My03Ljg1ODIzLDE4LjA0NDY1LDEuNzA0MjUsMzUuOTM3NDYsMy43NjY1Nyw1Mi4xNTksMTIuNjUyODYsNC4yOTQyNSwyLjM1MjQxLDguMDYyNzIsMS4wODk0OSwxMi4xNjgzLS4wNjgxLDMxLjA0ODU5LTguNzU0NCw2Mi41MzI3Mi0xNC42OTE0Niw5NC45NjgyNS0xNC4wMjQ1NiwyMS4zMTQuNDM4Miw0MS45ODQyNSwzLjg3ODkxLDYxLjY0Mzc5LDEyLjcwMjc2LDYuNDI3MjksMi44ODQ3OSw4LjgwODY5LDYuOTg2NDEsOC4yMTY1NywxMy44MS0xLjU5MzM4LDE4LjM2MDg0LTguOTMzNDUsMzQuODE3NTEtMTYuNTU1NzEsNTEuMTgwMzgtMi4wODM3NSw0LjQ3MzEzLTUuMTEzMiwxLjkwMDYzLTcuNjg0MTcuMjc2ODYtMjUuODcwMzMtMTYuMzM4ODYtNTMuNjUxLTI4LjcwMDYzLTgyLjI2NC0zOS4yNTUxNi0yMC4zMDkwNi03LjQ5MTQ2LTQwLjg0NzY4LTE0LjMwOTczLTYzLjA1ODIzLTE3LjcxMjkzLDYuODUzODgsNi43ODE4MiwxMy40NjQwNywxMy44MzI2NiwyMC42MDM1NSwyMC4yOTkxMiwzMS42MDk3OSwyOC42Myw2NC41NTE4NSw1NS40OTQsMTAyLjYyNDMsNzUuMzE3NjQsNC4yMDA1MiwyLjE4NzEyLDUuMjE5NDYsNC40Mjk0MiwyLjMzNzMyLDguNTIyMTMtMTAuNTgzNzQsMTUuMDI5MDgtMjEuNDEwODUsMjkuODczMzEtMzcuMjkwNjQsMzkuODIzMTItNC43MjI4MywyLjk1OTE4LTUuODUyNDYsNy4xNjgzOC03LjAyMjgzLDExLjg0MjItOS4xMDQzMywzNi4zNTkyOS0yMi4zMzIsNzAuOTU1ODEtNDQuMDI1MzEsMTAxLjg2NTA3YTE0MS4zMjU0NSwxNDEuMzI1NDUsMCwwLDEtMzIuNzkzMiwzMy44NDUyMWMtNS44MDIwOSw0LjE5ODExLTEwLjk2OTg0LDQuNjkyLTE2Ljg1MDguMjcxOC0xNC4xOTA0Ny0xMC42NjU4OS0yNC44NjAxOS0yNC40MDA5My0zNC43NTE4NC0zOC44NzctMi43MTgzNy0zLjk3ODMzLS43ODg4Ni02LjE1NjEyLDIuNzA4NC03Ljk1NSwyOC41MTE4My0xNC42NjUyNyw1My44NDQ5NS0zMy45MTk2Nyw3OC4yNjQzNi01NC40NTM3YTQ1Mi44NjksNDUyLjg2OSwwLDAsMCw0Mi41MjQwOS00MC4wNmMtMi44MDMzNy0yLjY3MTQxLTUuMjA3ODItLjcwNTI5LTcuMzA0OC0uMTQ2MTctNDcuMDUyLDEyLjU0NS05Mi4zNzI1NCwyOS4zODkyMS0xMzQuMDQwODIsNTUuMDEwOS01LjY5NTE1LDMuNTAxOTMtOC43MjM0NiwzLjA2NjE2LTExLjQ5OTg1LTMuMzM5MzctNy4wOTI2OC0xNi4zNjM2Ny0xMy42NjczNC0zMi44Njc1MS0xNC43NzQxOS01MC44NjkzNi0uMzYyNjgtNS44OTgxNC0zLjk2ODM2LTkuMDc1NTUtNy41NjI2NC0xMi41Njg0Ni0yNS42NTEyMy0yNC45MjcxNy00Ny45NjEtNTIuMzc0ODMtNjMuNDg0ODYtODQuODQzN0MxMi40MjgyOSw2OC4yNTUsNi44ODkxOSw1MS44MDAxNCw2LjIxMDg5LDMzLjkzMzgxYy0uMjY5LTcuMDg2MDYsMi42MDM2OC0xMS4wMzkyNSw5LjAwMTQ3LTEzLjM0NDlhMTY4LjI0NDkxLDE2OC4yNDQ5MSwwLDAsMSw0Ni4xNTYtOS42ODQxNGM2LjMyMDE1LS40MjE2Miw3Ljg3MzEsMS44MDQzMyw3LjcwNzY4LDcuNTA2NjgtMS4xNjg0MSw0MC4yODA2LDQuOTg5MjMsNzkuNzU1LDEzLjU3NjQ2LDExOC45MjMsMi40NjE1MiwxMS4yMjc0MSw1LjUyNDg2LDIyLjMyMjk0LDguMzEyMjcsMzMuNDc4ODRaTTE3OS40MTM2OSwzOS41MDhDMTY5LjA5Niw5Mi45NjE2MSwxMzYuNjc2ODUsMTMyLjkwNDIxLDEwMC4xMzIxMywxNzAuNTI4MmM0OS44OTE3Ny0yMC4zNzEsOTkuNjI4NTctMTEuMjM1NywxNDkuODI1NDMtMS4zMjA3NkMyMTEuNDQzODcsMTMzLjc1OTY5LDE5My45ODkyNiw4Ny43MTM2MywxNzkuNDEzNjksMzkuNTA4WiIvPjwvc3ZnPg==
[flyte-link]: https://docs.flyte.org

<details><summary>source</summary>
<p>

```markdown
[![flyte-badge]][flyte-link]

[flyte-badge]: https://img.shields.io/badge/flyte-carrier-gray.svg?color=7552A2&logo=data:image/svg%2bxml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB2aWV3Qm94PSIwIDAgMzYwIDMzMy40ODM3OSI+PGRlZnM+PHN0eWxlPi5jbHMtMXtmaWxsOnVybCgjbGluZWFyLWdyYWRpZW50KTt9PC9zdHlsZT48bGluZWFyR3JhZGllbnQgaWQ9ImxpbmVhci1ncmFkaWVudCIgeDE9IjYuMTkzNjUiIHkxPSIxNjcuMzQ3NDUiIHgyPSIzNTIiIHkyPSIxNjcuMzQ3NDUiIGdyYWRpZW50VW5pdHM9InVzZXJTcGFjZU9uVXNlIj48c3RvcCBvZmZzZXQ9IjAiIHN0b3AtY29sb3I9IiNlNWM5ZDMiLz48c3RvcCBvZmZzZXQ9IjAuMzMiIHN0b3AtY29sb3I9IiNiMzg1ZGYiLz48c3RvcCBvZmZzZXQ9IjEiIHN0b3AtY29sb3I9IiM2ZjJhZWYiLz48L2xpbmVhckdyYWRpZW50PjwvZGVmcz48cGF0aCBjbGFzcz0iY2xzLTEiIGQ9Ik05MC45NjQ3NSwxNzAuODEzMzRjNy42Mjc3Ny0xOC4xMjcsMTEuNTEwNDctMzYuNzc2NTUsMTUuMDA2NC01NS40OTI4OSw1Ljk2MDM3LTMxLjkxMDIsOS45OS02NC4wMzIzLDguNjM2MjEtOTYuNTk0NzUtLjI1NzIzLTYuMTg3NDIsMS40ODQ5NS04LjQ4NzYzLDguMTQ5My03Ljg1ODIzLDE4LjA0NDY1LDEuNzA0MjUsMzUuOTM3NDYsMy43NjY1Nyw1Mi4xNTksMTIuNjUyODYsNC4yOTQyNSwyLjM1MjQxLDguMDYyNzIsMS4wODk0OSwxMi4xNjgzLS4wNjgxLDMxLjA0ODU5LTguNzU0NCw2Mi41MzI3Mi0xNC42OTE0Niw5NC45NjgyNS0xNC4wMjQ1NiwyMS4zMTQuNDM4Miw0MS45ODQyNSwzLjg3ODkxLDYxLjY0Mzc5LDEyLjcwMjc2LDYuNDI3MjksMi44ODQ3OSw4LjgwODY5LDYuOTg2NDEsOC4yMTY1NywxMy44MS0xLjU5MzM4LDE4LjM2MDg0LTguOTMzNDUsMzQuODE3NTEtMTYuNTU1NzEsNTEuMTgwMzgtMi4wODM3NSw0LjQ3MzEzLTUuMTEzMiwxLjkwMDYzLTcuNjg0MTcuMjc2ODYtMjUuODcwMzMtMTYuMzM4ODYtNTMuNjUxLTI4LjcwMDYzLTgyLjI2NC0zOS4yNTUxNi0yMC4zMDkwNi03LjQ5MTQ2LTQwLjg0NzY4LTE0LjMwOTczLTYzLjA1ODIzLTE3LjcxMjkzLDYuODUzODgsNi43ODE4MiwxMy40NjQwNywxMy44MzI2NiwyMC42MDM1NSwyMC4yOTkxMiwzMS42MDk3OSwyOC42Myw2NC41NTE4NSw1NS40OTQsMTAyLjYyNDMsNzUuMzE3NjQsNC4yMDA1MiwyLjE4NzEyLDUuMjE5NDYsNC40Mjk0MiwyLjMzNzMyLDguNTIyMTMtMTAuNTgzNzQsMTUuMDI5MDgtMjEuNDEwODUsMjkuODczMzEtMzcuMjkwNjQsMzkuODIzMTItNC43MjI4MywyLjk1OTE4LTUuODUyNDYsNy4xNjgzOC03LjAyMjgzLDExLjg0MjItOS4xMDQzMywzNi4zNTkyOS0yMi4zMzIsNzAuOTU1ODEtNDQuMDI1MzEsMTAxLjg2NTA3YTE0MS4zMjU0NSwxNDEuMzI1NDUsMCwwLDEtMzIuNzkzMiwzMy44NDUyMWMtNS44MDIwOSw0LjE5ODExLTEwLjk2OTg0LDQuNjkyLTE2Ljg1MDguMjcxOC0xNC4xOTA0Ny0xMC42NjU4OS0yNC44NjAxOS0yNC40MDA5My0zNC43NTE4NC0zOC44NzctMi43MTgzNy0zLjk3ODMzLS43ODg4Ni02LjE1NjEyLDIuNzA4NC03Ljk1NSwyOC41MTE4My0xNC42NjUyNyw1My44NDQ5NS0zMy45MTk2Nyw3OC4yNjQzNi01NC40NTM3YTQ1Mi44NjksNDUyLjg2OSwwLDAsMCw0Mi41MjQwOS00MC4wNmMtMi44MDMzNy0yLjY3MTQxLTUuMjA3ODItLjcwNTI5LTcuMzA0OC0uMTQ2MTctNDcuMDUyLDEyLjU0NS05Mi4zNzI1NCwyOS4zODkyMS0xMzQuMDQwODIsNTUuMDEwOS01LjY5NTE1LDMuNTAxOTMtOC43MjM0NiwzLjA2NjE2LTExLjQ5OTg1LTMuMzM5MzctNy4wOTI2OC0xNi4zNjM2Ny0xMy42NjczNC0zMi44Njc1MS0xNC43NzQxOS01MC44NjkzNi0uMzYyNjgtNS44OTgxNC0zLjk2ODM2LTkuMDc1NTUtNy41NjI2NC0xMi41Njg0Ni0yNS42NTEyMy0yNC45MjcxNy00Ny45NjEtNTIuMzc0ODMtNjMuNDg0ODYtODQuODQzN0MxMi40MjgyOSw2OC4yNTUsNi44ODkxOSw1MS44MDAxNCw2LjIxMDg5LDMzLjkzMzgxYy0uMjY5LTcuMDg2MDYsMi42MDM2OC0xMS4wMzkyNSw5LjAwMTQ3LTEzLjM0NDlhMTY4LjI0NDkxLDE2OC4yNDQ5MSwwLDAsMSw0Ni4xNTYtOS42ODQxNGM2LjMyMDE1LS40MjE2Miw3Ljg3MzEsMS44MDQzMyw3LjcwNzY4LDcuNTA2NjgtMS4xNjg0MSw0MC4yODA2LDQuOTg5MjMsNzkuNzU1LDEzLjU3NjQ2LDExOC45MjMsMi40NjE1MiwxMS4yMjc0MSw1LjUyNDg2LDIyLjMyMjk0LDguMzEyMjcsMzMuNDc4ODRaTTE3OS40MTM2OSwzOS41MDhDMTY5LjA5Niw5Mi45NjE2MSwxMzYuNjc2ODUsMTMyLjkwNDIxLDEwMC4xMzIxMywxNzAuNTI4MmM0OS44OTE3Ny0yMC4zNzEsOTkuNjI4NTctMTEuMjM1NywxNDkuODI1NDMtMS4zMjA3NkMyMTEuNDQzODcsMTMzLjc1OTY5LDE5My45ODkyNiw4Ny43MTM2MywxNzkuNDEzNjksMzkuNTA4WiIvPjwvc3ZnPg==
[flyte-link]: https://docs.flyte.org
```

</p>
</details> 

Based on the [badges/shields docs](https://github.com/badges/shields/blob/master/doc/logos.md), as opposed to including the icon directly in the url, I believe a more widely adopted approach is to contribute it to [simple-icons](https://github.com/simple-icons/simple-icons). I do not know the details regarding the licensing in this case, but I believe https://github.com/lfai/artwork/blob/main/LICENSE.md and https://www.linuxfoundation.org/legal/trademark-usage are what applies.

**Comment by davidmirror-ops** [comment]
Hey @cameronraysmith thanks for taking the time and starting this discussion.

To make sure I understand: let's say there's a project/etc that uses Flyte in some way. The idea is to standardize a badge that they could add to their repo, pointing to the Flyte docs?

Sorry if I'm not entirely familiar with the idea of a "carrier" in this context :)

**Reply by cameronraysmith** [reply]
Hi @davidmirror-ops. Yes, that's exactly what I have in mind.

The use of `Carrier` is intended to be a bit of a joke riffing on the [aircraft carrier](https://en.wikipedia.org/wiki/Aircraft_carrier) or [airline carrier](https://en.wikipedia.org/w/index.php?search=%22airline+carrier%22&ns0=1) that probably has a potential to be confusing.

Maybe just `Flyte Workflows` like this

[![flyte-badge]][flyte-link]

[flyte-badge]: https://img.shields.io/badge/Flyte-Workflows-gray.svg?color=7552A2&logo=data:image/svg%2bxml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB2aWV3Qm94PSIwIDAgMzYwIDMzMy40ODM3OSI+PGRlZnM+PHN0eWxlPi5jbHMtMXtmaWxsOnVybCgjbGluZWFyLWdyYWRpZW50KTt9PC9zdHlsZT48bGluZWFyR3JhZGllbnQgaWQ9ImxpbmVhci1ncmFkaWVudCIgeDE9IjYuMTkzNjUiIHkxPSIxNjcuMzQ3NDUiIHgyPSIzNTIiIHkyPSIxNjcuMzQ3NDUiIGdyYWRpZW50VW5pdHM9InVzZXJTcGFjZU9uVXNlIj48c3RvcCBvZmZzZXQ9IjAiIHN0b3AtY29sb3I9IiNlNWM5ZDMiLz48c3RvcCBvZmZzZXQ9IjAuMzMiIHN0b3AtY29sb3I9IiNiMzg1ZGYiLz48c3RvcCBvZmZzZXQ9IjEiIHN0b3AtY29sb3I9IiM2ZjJhZWYiLz48L2xpbmVhckdyYWRpZW50PjwvZGVmcz48cGF0aCBjbGFzcz0iY2xzLTEiIGQ9Ik05MC45NjQ3NSwxNzAuODEzMzRjNy42Mjc3Ny0xOC4xMjcsMTEuNTEwNDctMzYuNzc2NTUsMTUuMDA2NC01NS40OTI4OSw1Ljk2MDM3LTMxLjkxMDIsOS45OS02NC4wMzIzLDguNjM2MjEtOTYuNTk0NzUtLjI1NzIzLTYuMTg3NDIsMS40ODQ5NS04LjQ4NzYzLDguMTQ5My03Ljg1ODIzLDE4LjA0NDY1LDEuNzA0MjUsMzUuOTM3NDYsMy43NjY1Nyw1Mi4xNTksMTIuNjUyODYsNC4yOTQyNSwyLjM1MjQxLDguMDYyNzIsMS4wODk0OSwxMi4xNjgzLS4wNjgxLDMxLjA0ODU5LTguNzU0NCw2Mi41MzI3Mi0xNC42OTE0Niw5NC45NjgyNS0xNC4wMjQ1NiwyMS4zMTQuNDM4Miw0MS45ODQyNSwzLjg3ODkxLDYxLjY0Mzc5LDEyLjcwMjc2LDYuNDI3MjksMi44ODQ3OSw4LjgwODY5LDYuOTg2NDEsOC4yMTY1NywxMy44MS0xLjU5MzM4LDE4LjM2MDg0LTguOTMzNDUsMzQuODE3NTEtMTYuNTU1NzEsNTEuMTgwMzgtMi4wODM3NSw0LjQ3MzEzLTUuMTEzMiwxLjkwMDYzLTcuNjg0MTcuMjc2ODYtMjUuODcwMzMtMTYuMzM4ODYtNTMuNjUxLTI4LjcwMDYzLTgyLjI2NC0zOS4yNTUxNi0yMC4zMDkwNi03LjQ5MTQ2LTQwLjg0NzY4LTE0LjMwOTczLTYzLjA1ODIzLTE3LjcxMjkzLDYuODUzODgsNi43ODE4MiwxMy40NjQwNywxMy44MzI2NiwyMC42MDM1NSwyMC4yOTkxMiwzMS42MDk3OSwyOC42Myw2NC41NTE4NSw1NS40OTQsMTAyLjYyNDMsNzUuMzE3NjQsNC4yMDA1MiwyLjE4NzEyLDUuMjE5NDYsNC40Mjk0MiwyLjMzNzMyLDguNTIyMTMtMTAuNTgzNzQsMTUuMDI5MDgtMjEuNDEwODUsMjkuODczMzEtMzcuMjkwNjQsMzkuODIzMTItNC43MjI4MywyLjk1OTE4LTUuODUyNDYsNy4xNjgzOC03LjAyMjgzLDExLjg0MjItOS4xMDQzMywzNi4zNTkyOS0yMi4zMzIsNzAuOTU1ODEtNDQuMDI1MzEsMTAxLjg2NTA3YTE0MS4zMjU0NSwxNDEuMzI1NDUsMCwwLDEtMzIuNzkzMiwzMy44NDUyMWMtNS44MDIwOSw0LjE5ODExLTEwLjk2OTg0LDQuNjkyLTE2Ljg1MDguMjcxOC0xNC4xOTA0Ny0xMC42NjU4OS0yNC44NjAxOS0yNC40MDA5My0zNC43NTE4NC0zOC44NzctMi43MTgzNy0zLjk3ODMzLS43ODg4Ni02LjE1NjEyLDIuNzA4NC03Ljk1NSwyOC41MTE4My0xNC42NjUyNyw1My44NDQ5NS0zMy45MTk2Nyw3OC4yNjQzNi01NC40NTM3YTQ1Mi44NjksNDUyLjg2OSwwLDAsMCw0Mi41MjQwOS00MC4wNmMtMi44MDMzNy0yLjY3MTQxLTUuMjA3ODItLjcwNTI5LTcuMzA0OC0uMTQ2MTctNDcuMDUyLDEyLjU0NS05Mi4zNzI1NCwyOS4zODkyMS0xMzQuMDQwODIsNTUuMDEwOS01LjY5NTE1LDMuNTAxOTMtOC43MjM0NiwzLjA2NjE2LTExLjQ5OTg1LTMuMzM5MzctNy4wOTI2OC0xNi4zNjM2Ny0xMy42NjczNC0zMi44Njc1MS0xNC43NzQxOS01MC44NjkzNi0uMzYyNjgtNS44OTgxNC0zLjk2ODM2LTkuMDc1NTUtNy41NjI2NC0xMi41Njg0Ni0yNS42NTEyMy0yNC45MjcxNy00Ny45NjEtNTIuMzc0ODMtNjMuNDg0ODYtODQuODQzN0MxMi40MjgyOSw2OC4yNTUsNi44ODkxOSw1MS44MDAxNCw2LjIxMDg5LDMzLjkzMzgxYy0uMjY5LTcuMDg2MDYsMi42MDM2OC0xMS4wMzkyNSw5LjAwMTQ3LTEzLjM0NDlhMTY4LjI0NDkxLDE2OC4yNDQ5MSwwLDAsMSw0Ni4xNTYtOS42ODQxNGM2LjMyMDE1LS40MjE2Miw3Ljg3MzEsMS44MDQzMyw3LjcwNzY4LDcuNTA2NjgtMS4xNjg0MSw0MC4yODA2LDQuOTg5MjMsNzkuNzU1LDEzLjU3NjQ2LDExOC45MjMsMi40NjE1MiwxMS4yMjc0MSw1LjUyNDg2LDIyLjMyMjk0LDguMzEyMjcsMzMuNDc4ODRaTTE3OS40MTM2OSwzOS41MDhDMTY5LjA5Niw5Mi45NjE2MSwxMzYuNjc2ODUsMTMyLjkwNDIxLDEwMC4xMzIxMywxNzAuNTI4MmM0OS44OTE3Ny0yMC4zNzEsOTkuNjI4NTctMTEuMjM1NywxNDkuODI1NDMtMS4zMjA3NkMyMTEuNDQzODcsMTMzLjc1OTY5LDE5My45ODkyNiw4Ny43MTM2MywxNzkuNDEzNjksMzkuNTA4WiIvPjwvc3ZnPg==
[flyte-link]: https://docs.flyte.org

or something else would be better.

**Reply by davidmirror-ops** [reply]
Awesome @cameronraysmith! I think `workflows` is straightforward, but will let other community members give their opinions cc @fg91 @bstadlbauer et.al.

**Reply by fg91** [reply]
I slightly prefer "workflows" because one needs to know about Flyte to get the joke ^^

**Comment by kumare3** [comment]
@cameronraysmith I love this idea. I actually want to standardize a way to let people declare workflows where-ever that can be run on any flyte cluster, any cloud. 🚀

**Comment by bstadlbauer** [comment]
@cameronraysmith Love it! For me both "Carrier" as well as "Workflows" work 👍 
Not 100% sure what licensing applies, but if we're allowed to, feel free to contribute to simple icons!
